src/mlpack/methods/lsh/lsh_search_impl.hpp

```diff
@@ -733,7 +762,8 @@ void LSHSearch<SortPolicy>::ReturnIndicesFromTable(
 
     // Retrieve candidates.
     size_t start = 0;
-    for (size_t i = 0; i < numTablesToSearch; ++i) // For all tables
+
+    for (long long int i = 0; i < numTablesToSearch; ++i) // For all tables
```

Comment:
That's pretty reasonable. I can go back to using `size_t` and ask to detect version 3 and up.